pkg/config/remote/service/util.go

```diff
@@ -69,9 +69,9 @@ func buildLatestConfigsRequest(hostname string, state uptane.State, activeClient
 		AgentVersion:                 version.AgentVersion,
 		Products:                     data.ProductListToString(productsList),
 		NewProducts:                  data.ProductListToString(newProductsList),
-		CurrentConfigSnapshotVersion: state.ConfigSnapshotVersion(),
-		CurrentConfigRootVersion:     state.ConfigRootVersion(),
-		CurrentDirectorRootVersion:   state.DirectorRootVersion(),
+		CurrentConfigSnapshotVersion: state.ConfigSnapshot,
+		CurrentConfigRootVersion:     state.ConfigRoot,
+		CurrentDirectorRootVersion:   state.DirectorRoot,
 		ActiveClients:                activeClients,
 		BackendClientState:           clientState,
```

Comment:
Just to make sure I understand: this `pbgo.LatestConfigsRequest` object we are building is the request that the agent sends to the backend to poll new remote config updates, right? And it communicates the current versions it has so that the backend can adapt its response, right?

Reply:
It's an arbitrary field the backend controls to store state and efficiently send updates. Today it keeps a bunch of hashes to compare what the agent has vs what he should have